packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -57,6 +57,7 @@
                         :has-error="isMobileNumberEmpty || isMobileNumberInvalid"
                         aria-describedby="mobile-number-error"
                         :aria-invalid="!isMobileNumberValid"
```

Comment:
Do you have a console error that `isMobileNumberValid is not defined on the instance but referenced during render`. Maybe while you are here you can fix this one? I checked that the value should be `isMobileNumberInvalid` and not `!isMobileNumberValid`